src/tape_drivers/tape_drivers.h

```diff
@@ -150,12 +150,11 @@ static inline int _sense2errorcode(uint32_t sense, struct error_table *table, ch
 struct supported_device {
 	char vendor_id[VENDOR_ID_LENGTH + 1];
 	char product_id[PRODUCT_ID_LENGTH + 1];
-	int  vendor_type;
 	int  drive_type;
 	char product_name[PRODUCT_NAME_LENGTH + 1];
 };
 
-#define TAPEDRIVE(v, p, vt, dt, n) &(struct supported_device){ v, p, vt, dt, n }
+#define TAPEDRIVE(v, p, t, n) &(struct supported_device){ v, p, t, n }
```

Comment:
Please don't corrupt #296.

Could you explain why you need to remove `vendor_type` field if you are feeling this is required ?